optional_plugins/runner_remote/setup.py

```diff
@@ -13,12 +13,13 @@
 # Copyright: Red Hat Inc. 2017
 # Author: Cleber Rosa <crosa@redhat.com>
 
-import sys
+import distro
```

Comment:
Nice to meet you `distro`! But, there's also `avocado.utils.distro` which should be sufficient here.